### PR TITLE
Update/end ftk

### PR DESCRIPTION
**Pull request by whaeck**
Updated resonance reconstruction using the new mf2 interface in anticipation of the lrf7 extension.

I'm currently running tests again.

### CODE REVIEW

File: src/resonanceReconstruction/breitWigner/multilevel/Apply/test/Apply.test.cpp [hunk 1]
```diff
@@ -21,10 +21,13 @@ void ignore( T&& ){}
...
 SCENARIO( "Integration test" ){
   const auto Na22 = resonances();
-  const auto& isotope = Na22.isotopes.front();
-  const auto& energyRange = isotope.energyRanges().front();
-  const auto& mlbw = std::get< 2 >( energyRange );
-  const auto type = Apply().build( mlbw, channelRadius( 22. ), radius( mlbw.AP() ) );
+  const auto& isotope = Na22.isotopes().front();
+  const auto& resonanceRange = isotope.resonanceRanges().front();
+  EnergyRange energyRange{ resonanceRange.EL() * electronVolts,
+                           resonanceRange.EH() * electronVolts }
```
Comment (jlconlin) [inline src/resonanceReconstruction/breitWigner/multilevel/Apply/test/Apply.test.cpp:27]: This line has a missing semi-colon.

File: src/resonanceReconstruction/reichMoore/Apply/src/call.hpp [hunk 2]
```diff
@@ -1,34 +1,56 @@
...
+                                radius( rm.AP() ),
+                                radius( range.scatteringRadius().value() ),
+                                true ) );
+      }
     }
   }
+  catch ( ... ) {
+
+    throw std::runtime_error( "The resonance range does not appear to contain "
+                              "SLBW parameters" );
```
Comment (jlconlin) [inline src/resonanceReconstruction/reichMoore/Apply/src/call.hpp:54]: SLBW should be Reich-Moore.

File: src/resonanceReconstruction/breitWigner/singleLevel/Type/test/Type.test.cpp [hunk 3]
```diff
@@ -22,11 +22,14 @@ const std::vector< double >& capture();
...
 SCENARIO( "Integration test" ){
   const auto Rh105 = resonances();
-  const auto& isotope = Rh105.isotopes.front();
-  const auto& energyRange = isotope.energyRanges().front();
-  const auto& slbw = std::get< 1 >( energyRange );
+  const auto& isotope = Rh105.isotopes().front();
+  const auto& resonanceRange = isotope.resonanceRanges().front();
+  EnergyRange energyRange{ resonanceRange.EL() * electronVolts,
+                           resonanceRange.EH() * electronVolts }
+  const auto& slbw = std::get< 1 >( resonanceRange );
```
Comment (jlconlin) [inline src/resonanceReconstruction/breitWigner/singleLevel/Type/test/Type.test.cpp:29]: This line doesn't compile.

Comment (whaeck) [reply src/resonanceReconstruction/breitWigner/singleLevel/Type/test/Type.test.cpp]: Should be fixed now

**Comment by jlconlin** [comment]
I've made several additional changes. Everything compiles and looks good now.

**Pull request merged**